projects/observability/src/shared/graphql/request/handlers/entities/query/entities-graphql-query-builder.service.ts

```diff
@@ -105,7 +106,8 @@ export interface GraphQlEntitiesRequest {
   filters?: GraphQlFilter[];
   includeTotal?: boolean;
   includeInactive?: boolean;
-  ignoreGlobalFilters?: boolean;
+  ignoreApiDiscoveryStateFilter?: boolean;
+  ignoreGlobalEnvironmentFilter?: boolean;
```

Comment:
Curious what the use case is to ignore the global environment filter - that sounds like a big on the caller side.

Reply:
purpose of this is on some of the config pages which allow for environment selection. We want subsequent API/service selection in such workflows to be limited by the environment choice the user made.